cfme/physical/provider/__init__.py

```diff
@@ -133,6 +133,8 @@ class Details(CFMENavigateStep):
 
     def step(self):
         self.prerequisite_view.entities.get_entity(name=self.obj.name, surf_pages=True).click()
+        if self.view.toolbar.view_selector.is_displayed:
+            self.view.toolbar.view_selector.select('Summary View')
```

Comment:
Yeah I suppose we can mark Summary view as `Details` and later when we need Dashboard view we implement another class

```python 
@navigator.register(PhysicalProvider, 'DetailsDashboard')
class DetailsDashboard(CFMENavigateStep):
   ...
```

Right @mshriver ?